File: src/test/resources/org/jetbrains/research/ml/dataset/anonymizer/transformations/anonymization/java/data/classes_and_methods/in_3_overriding_with_variables.java
public abstract class Test3_0 {

    @Override
    public boolean equals(Object obj) {
        return super.equals(obj);
    }

    public static void main() {
    }

    public int calculate(int x, int y) {
        return 0;
    }

    abstract public void test(int x, int y);
}

class Test3_1 extends Test3_0 {

    @Override
    public void test(int x, int y) {

    }
}

interface Operationable{
    int calculate(int x, int y);
}

File: src/test/resources/org/jetbrains/research/ml/dataset/anonymizer/transformations/anonymization/kotlin/data/classes_and_methods/in_3_overriding_with_variables.kt
public abstract class Test3_0 : Operationable {

    override fun equals(other: Any?): Boolean {
        return super.equals(other)
    }

    fun main() {

    }


    override fun calculate(x: Int, y: Int): Int {
        return 0;
    }

    abstract fun test(x: Int, y: Int)
}

class Test3_1 : Test3_0 {
    override fun test(x: Int, y: Int) {

    }
}

interface Operationable {
    fun calculate(x: Int, y: Int): Int
}
